source/community/release_notes/palm.rst

```diff
@@ -15,30 +15,241 @@ These are the release notes for the Palm release, the 16th community release of
+* `student.enable_enrollment_confirmation_email <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/student/toggles.py#L61>`_
+* `student.enable_fallback_recommendations <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/student/toggles.py#L26>`_
+* `third_party_auth.apple_user_migration <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/common/djangoapps/third_party_auth/toggles.py#L9>`_
+* `video_config.public_video_share <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/openedx/core/djangoapps/video_config/toggles.py#L8>`_
+
+The following Features/Toggles were removed:
+
+* `block_structure.raise_error_when_not_found <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/openedx/core/djangoapps/content/block_structure/config/__init__.py#L51>`_
+* `credentials.use_learner_record_mfe <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/openedx/core/djangoapps/credentials/config.py#L10>`_
+
+The following new settings were added:
+
+* `SECURITY_PAGE_URL <https://github.com/openedx/edx-platform/blob/8e6e2997151e010276f5d76db7269c1f1d2af702/lms/envs/common.py#L3355)>`_
+
+The following settings were removed:
+
+* `LEARNER_RECORD_MFE_URL <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/lms/envs/common.py#L4894>`_
+* `XBLOCK_SELECT_FUNCTION <https://github.com/openedx/edx-platform/blob/db111c05f6d8172c5629e9d723844565ac657476/lms/envs/common.py#L1531>`_
+
+
 
 Deprecations & Removals
 ***********************
 
+The Django setting :code:`JWT_AUTH.JWT_PRIVATE_SIGNING_JWK` can be removed from CMS configs if it is still present, as it has only been used by the LMS for some time now.
+
 
 Developer Experience
 ********************
 
+API changes
+===========
+
+* Filter by active courses in the course listing API.
+* Added :code:`certificate_available_date` to course detail API.
+
+New filters and hook events
+===========================
+
+* Added :code:`VerticalBlockChildRenderStarted` and :code:`VerticalBlockRenderCompleted` filters that will be called at the start of rendering a vertical and after rendering is completed.
+* Added :code:`AccountSettingsRenderStarted` filter which passes the account settings context before is rendered.
+* Added events :code:`XBLOCK_PUBLISHED`, :code:`XBLOCK_DELETED`, and :code:`XBLOCK_DUPLICATED` to signal changes to xblocks in taxonomy-connector/course-discovery.
+
 
 Researcher & Data Experiences
 *****************************
 
+* Tracking events have been added for reporting of threads, responses and comments, and for marking responses as endorsed. 
+
```

Comment:
@bmtcril I found these news events that were added going through the palm commits. Is this accurate? Is it complete?

Reply:
I wasn't aware of them, but I was speaking with @ayub02 about the existing events being maintained (they are). He might know more about the new events. We should document them here, too: https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_log_event_list.html

Reply:
Hi! The new discussion related events can be found [here](https://docs.google.com/spreadsheets/d/1YoOWlodw6lutfI-8pDZDRIOqjo6J9D5L2SuENn-mXmQ/edit?usp=sharing). I'll be on vacation from 19th June till mid July and won't be able to update the docs, until after. Feel free to use the sheet to update the docs if you want.

Reply:
Hi @ayub02 I've taken an initial stab [here](https://github.com/openedx/docs.openedx.org/pull/343), but I'm not super clear on the spreadsheet meanings or how up to date it is (looks like there are things in the actual columns which are not matching what's expected). I haven't gone through every tab, but it looks like at least the fields on search also changed?

Is the intention that what is in the "actual" column matches what's in the code right now for every event on every tab? I can take a more thorough pass if that's the case.

Reply:
@bmtcril Correct. The "Actual" column is what's in the code (at least as per my testing). There are differences between actual and intended, but they don't need to be addressed on priority, in my opinion.